themes/red-theme/lms/templates/schedules/edx_ace/recurringnudge_day3/email/body.txt

```diff
@@ -4,14 +4,14 @@
 This is the RED theme!
 {% if course_ids|length > 1 %}
 {% blocktrans trimmed %}
-    Remember when you enrolled in {{ course_name }}, and other courses on edX.org? We do, and we’re glad
+    Remember when you enrolled in {{ course_name }}, and other courses on {{ platform_name }}? We do, and we’re glad
```

Comment:
This is a template used in an example theme. I changed to using {{ platform_name }} to match exactly other usages of this string, to simplify translator burden.